services/ui-src/src/utils/reports/entities.ts

```diff
@@ -18,10 +17,12 @@ const getRepeatedField = (
     for (const [key, value] of Object.entries(entity)) {
       if (key.includes(repeatedKey) && value) {
         const id = key.replace(repeatedKey, "").split("Q");
-        const displayValue =
-          repeatedKey === "fundingSources_quarters"
-            ? `$${value}`
-            : convertToThousandsSeparatedString(value).maskedValue;
+        let displayValue = `${value}`;
+
+        if (repeatedKey === "fundingSources_quarters") {
+          displayValue = `$${value}`;
+        }
```

Comment:
This version removes thousands separators from `quarterlyProjections`, `objectiveTargets_projections_`, and `objectiveTargets_actual_`. Is it intended to affect all three, or do we still need a special case for `quarterlyProjections`?

Reply:
I believe the intention is to remove it from all of them. I believe the `objectiveTargets` fields are the SAR equivalents of `quarterlyProjections`? But either way, if it has to do with the objectives, product has been clear that we are just going to let the user enter _whatever_.

[Slack convo here](https://coforma.slack.com/archives/C06N0V30TM4/p1714147864925089), for reference